app/uk/gov/hmrc/gform/gform/FormController.scala

```diff
@@ -358,11 +358,23 @@ class FormController(
       implicit request => implicit l => cache => _ => formModelOptics =>
         def processEditAddToList(processData: ProcessData, idx: Int, addToListId: AddToListId): Future[Result] = {
 
-          val addToListIteration =
-            processData.formModelOptics.formModelVisibilityOptics.formModel.brackets.addToListById(addToListId, idx)
+          val brackets: BracketsWithSectionNumber[Visibility] =
```

Comment:
I don't think switching to visibility model was a good decision.

Note that `"defaultPage"` can have an `"includeIf"` so in Visibility model `addToListIteration.secondSectionNumber` may not succeed.